source/faq.txt

```diff
@@ -200,6 +200,18 @@ this only works if the operating system supports ``SO_KEEPALIVE``\ .
 .. warning::
     If a firewall ignores or drops the ``keepAlive`` packets this may not work
 
+.. warning::
+    This option is deprecated in 5.2.0 and will be removed in a future release.
```

Comment:
let's wait to do this until we make a decision on https://github.com/mongodb/node-mongodb-native/pull/3621 and the corresponding ticket; it may end up being easier to track via a DOCSP ticket instead